src/core/oned/MultiFormatOneDReader.ts

```diff
@@ -39,6 +40,9 @@ export default class MultiFormatOneDReader extends OneDReader {
         const useCode39CheckDigit = hints && hints.get(DecodeHintType.ASSUME_CODE_39_CHECK_DIGIT) !== undefined;
 
         if (possibleFormats) {
+            if (possibleFormats.get(BarcodeFormat.EAN_13)) {
```

Comment:
What about EAN-8 and UPC-A here? Looks like your code supports those as well?

Reply:
EAN-8 and UPC-A are not yet supported, if you take a look: https://github.com/pimuzzo/library/blob/244bba34311f500dad77ea69659b05381352c89c/src/core/oned/MultiFormatUPCEANReader.ts there are some TODOs.
Maybe for those I will do another pull request after this

Reply:
That's great, It's easier to handle different ports in separated PRs.